async function a() {
    try {
        const a = await done();
        console.log(10);
    } catch(e) {
        return 34;
    }

    try {

    } catch(e) {
        console.log('teste')
        return 2;
    }
}

function a() {
    try {
        console.log('teste')
    } catch (e) {
        console.log('teste')
        console.log('teste')
        console.log('teste')
        console.log('teste')
        console.log('teste')
        console.log('teste')
    }

    try{

    }catch(e){
        return 3
    }
}